bigframes/operations/aggregations.py

```diff
@@ -78,6 +83,11 @@ def name(self) -> str:
     def arguments(self) -> int:
         ...
 
+    @property
+    def order_independent(self):
+        # Almost all aggregation functions are order independent, excepting array and string agg
```

Comment:
Let's make this a docstring too.

```suggestion
        """True if results don't depend on the order of the input.

        Almost all aggregation functions are order independent, excepting ``array_agg`` and ``string_agg``.
        """
```

Reply:
done